apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep1.vue

```diff
@@ -142,6 +142,7 @@ watch([() => collectorFormState.provider, () => state.selectedRepository], async
 }, { immediate: true });
 
 onMounted(() => {
+    collectorFormStore.resetForm();
```

Comment:
question: Is `onMounted` hook always executed?

Reply:
Yes, It is!